blogs/2025/07/15/copilot-coding-agent.md

```diff
@@ -0,0 +1,88 @@
+
+
+# Command GitHub's Coding Agent from VS Code
+
+Today, we’re excited to give you a first look at the Copilot Coding Agent in Visual Studio Code.
+
+Instead of only having one agent in VS Code, you can now have multiple agents running at once working on your behalf. This is _actually_ being a 10x developer. It's officially a a thing. You get to hand off any work you need done to to a squad of AI teammates.
```

Comment:
nit: maybe mention one `synchronous` or `local` agent?

Reply:
like this, updated!